fixing layout for button

championshuttler opened the pull request:
Fixes #2056 
Before: 
![screenshot_2018-09-13 lastpass add-ons fur android](https://user-images.githubusercontent.com/22130317/45481547-7825ea80-b769-11e8-9db8-c413866599ff.png)

After: 
![screenshot_2018-09-13 lastpass add-ons fur android 1](https://user-images.githubusercontent.com/22130317/45481554-7d833500-b769-11e8-9a85-84f5a6d823d5.png)

bobsilverberg commented:
This looks good, thanks @championshuttler. While testing I noticed that the button below this has a similar issue and therefore could also benefit from an `auto` height. Would you mind fixing that as well?

![screenshot 2018-09-13 14 10 07](https://user-images.githubusercontent.com/142755/45507004-e87b3e80-b75e-11e8-91ef-b60e6d3309bb.png)

championshuttler commented on src/ui/components/Button/styles.scss:
```diff
@@ -63,7 +63,7 @@ $puffy-font-size: 15px;
       &:link {
         font-size: $micro-font-size;
-        height: 24px;
+        height: auto;
```
Done :D

championshuttler commented:
For the second button. I added min-height - 48px and changed height to auto.

Before
![screenshot_2018-09-14 lastpass add-ons fur android](https://user-images.githubusercontent.com/22130317/45510177-14c2a980-b7b7-11e8-9db8-037cb60bf2ad.png)

After
![screenshot_2018-09-14 lastpass add-ons fur android 1](https://user-images.githubusercontent.com/22130317/45510183-1a1ff400-b7b7-11e8-9783-f946a920f062.png)

willdurand commented:
@championshuttler in a follow-up, could you add some padding maybe? I find the blue button hard to read.
